modules/opentelemetry/src/main/scala/zio/opentelemetry/OpenTelemetry.scala

```diff
@@ -0,0 +1,81 @@
+package zio.opentelemetry
+
+import java.util.concurrent.TimeUnit
+
+import io.opentelemetry.trace._
+import zio.clock.Clock
+import zio.{ FiberRef, UIO, URIO, URLayer, ZIO, ZLayer }
+import zio.opentelemetry.SpanUtils.endSpan
+
+object OpenTelemetry {
+  trait Service {
+    val currentSpan: FiberRef[Span]
+    def createRoot(spanName: String, spanKind: Span.Kind): UIO[Span]
+    def createChildOf(spanName: String, parent: Span, spanKind: Span.Kind): UIO[Span]
+    def createChild(spanName: String, spanKind: Span.Kind): UIO[Span]
```

Comment:
This probably does not belong here and should go to `CurrentSpan`, as it is not an elementary operation build on `Tracer`